CHANGELOG.md

```diff
@@ -101,6 +101,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * Editing a hidden commit now makes it visible.
 
+* `jj split` will no longer reference the paths given on the command line when
+  none have been given in interactive mode.
```

Comment:
I don't think this needs to be mentioned. I can't think of a user group who would care to learn this. It also doesn't seem like it should be considered a fixed bug.